Blish HUD/GameServices/GameIntegrationService.cs

```diff
@@ -341,7 +343,21 @@ public class GameIntegrationService : GameService {
                         break;
 
                     case WindowUtil.OverlayUpdateResponse.Errored:
-                        this.Gw2Process = null;
+                        switch (updateResult.ErrorCode) {
+                            case 1400:
+                                this.Gw2Process.Refresh();
+
+                                if (this.Gw2Process.MainWindowHandle == IntPtr.Zero) {
+                                    // Guild Wars 2 most likely closed
+                                    goto case -1;
+                                }
+
```

Comment:
This shouldn't work. I believe the user was just lucky when they tested this. Either `Gw2WindowHandle` has to be set here again, or `Gw2Process` needs to be reset here again, or `Gw2WindowHandle` should be modified to point to `Gw2Process.MainWindowHandle`.

Reply:
Good catch.  Updated Gw2WindowHandle to point to `Gw2Process.MainWindowHandle`.